health/health.go

```diff
@@ -263,6 +278,9 @@ func overallErrorLocked() error {
 	_ = lastMapRequestHeard
 
 	var errs []error
+	if !receiveIPv4Running || !receiveDERPRunning {
```

Comment:
break this into 3 separate error cases for clarity?

and then you'd need to know from magicsock whether IPv6 was wanted.